### ISSUE TITLE
[Legitimate Site Blocked] binances.club

### ISSUE THREAD

**Issue by faramsokol** [MetaMask/eth-phishing-detect#7870]
Hi. In thread https://github.com/MetaMask/eth-phishing-detect/issues/6696, no one answered me on the issue !!

**Comment by raomohsin39** [comment]
Admin, Hello! Please unblock this site

**Comment by devenblake** [comment]
Duplicate of #6696.
